libraries/glad/include/glad/khrplatform.h

```diff
@@ -96,13 +96,13 @@
  * This precedes the return type of the function in the function prototype.
```

Comment:
this this file should be moved to 'libraries/glad' folder

Reply:
please write here about this solution

Reply:
Tried do this and add khrplatform.h with PRIVATE key. Cmake built the project, but I couldn't include this file into glad.h. It worked only with PUBLIC key. So it will be similar with current folder structure. It would be better if these files are located in the same directory.